boa/src/object/gcobject.rs

```diff
@@ -26,17 +26,19 @@ use std::{
 };
 
 /// A wrapper type for an immutably borrowed `Object`.
-pub type Ref<'object> = GcCellRef<'object, Object>;
+pub type Ref<'object, T> = GcCellRef<'object, T>;
 
 /// A wrapper type for a mutably borrowed `Object`.
-pub type RefMut<'object> = GcCellRefMut<'object, Object>;
+pub type RefMut<'object, T> = GcCellRefMut<'object, T>;
```

Comment:
You changed `Ref` and `RefMut` to be generic but the comments and the lifetime name still mention `Object` and `'object`.